Gcd/gcd.cpp

```diff
@@ -0,0 +1,12 @@
+#include <bits/stdc++.h>
+using namespace std;
+int main()
+{
+  ios_base::sync_with_stdio(0);
```

Comment:
Please remove this. Use C++ standard cin and cout. This is confusing for learning purposes.